internal/profiles/models/models.go

```diff
@@ -15,6 +15,75 @@
 // Package models contains domain models for profiles
 package models
 
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/google/uuid"
+
+	"github.com/stacklok/minder/internal/db"
+	minderv1 "github.com/stacklok/minder/pkg/api/protobuf/go/minder/v1"
+)
+
+// ProfileAggregate represents a profile along with rule instances
+type ProfileAggregate struct {
```

Comment:
These new types are intended to address the fact that neither the protobuf or the database schema representations of `Profile` match up neatly with what a lot of the code in minder actually needs. These will be used elsewhere in the codebase in future.